test/model/unreads_test.dart

```diff
@@ -469,6 +469,145 @@ void main() {
         }
       }
     });
+
+    group('moves', () {
+      final origChannel = eg.stream();
+      const origTopic = 'origTopic';
+      const newTopic = 'newTopic';
+
+      final readMessages = List<StreamMessage>.generate(10,
+        (_) => eg.streamMessage(
```

Comment:
Messages can get mutated to handle events. So let's have these get generated fresh by each test that needs them.

(In particular if we refactored so that instead of calling `Unreads.handleUpdateMessageEvent` like other tests in this file, these called `PerAccountStore.handleEvent` like we do in some other areas of test/model/, I believe these would break as written.)